app/assets/icons/i-plus.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="22px" height="22px" viewBox="0 0 22 22" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 49.3 (51167) - http://www.bohemiancoding.com/sketch -->
+    <title>icons/i-plus--blue</title>
+    <desc>Created with Sketch.</desc>
```

Comment:
- [ ] Include the current develop branch, then you get the updated icon-set, the naming of the icon has changed and I updated the icon (removed the title and desc attribute)